controllers/kubekey/cluster_controller.go

```diff
@@ -113,7 +114,7 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	}
 
 	// Check if the configMap already exists
-	if err := r.Get(ctx, types.NamespacedName{Name: cluster.Name, Namespace: "kubekey-system"}, cmFound); err == nil {
+	if err := r.Get(ctx, types.NamespacedName{Name: cluster.Name, Namespace: r.Namespace}, cmFound); err == nil {
```

Comment:
Hi, I think we can use `req.NamespacedName`. Because this configmap and the cluster object are always in the same namespace.

Reply:
Got it yeah I see we are using the namespace scoped role. Makes sense to me. I have update the code please take a look.